helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl

```diff
@@ -36,20 +36,20 @@ env:
     value: {{ .Values.evidenceApi.clientId }}
   - name: EVIDENCE_API_OAUTH_CLIENT_SECRET
     value: {{ .Values.evidenceApi.clientSecret }}
-  - name: CLOUD_PLATFORM_QUEUE_REGION
-    value: {{ .Values.cloud-platform.aws.sqs.region }}
-  - name: CLOUD_PLATFORM_QUEUE_ACCESS_KEY
-    value: {{ .Values.cloud-platform.aws.sqs.accessKey }}
-  - name: CLOUD_PLATFORM_QUEUE_SECRET_KEY
-    value: {{ .Values.cloud-platform.aws.sqs.secretKey }}
+  - name: cloudPlatform_QUEUE_REGION
```

Comment:
name should be CLOUD_PLATFORM_QUEUE_REGION and should sync with the docker-compose.override.yml